docker/install_dependencies.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# exit on errors
+set -e
+
+# Get ubuntu distribution code name. All STOL APT debian packages are pushed to S3 bucket based on distribution codename.
+. /etc/lsb-release
+
+# add the STOL APT repository
+echo "deb [trusted=yes] http://s3.amazonaws.com/stol-apt-repository focal main" | sudo tee /etc/apt/sources.list.d/stol-apt-repository.list
+
+#${DISTRIB_CODENAME}
+
+sudo apt-get update
+
+# install all things needed for deployment, always done
```

Comment:
This comment can be updated since we are only installing the carma-j2735 package. It can instead say what that library is and why its required